Seneca-CDOT/starchart
# Setup Chakra UI

Fixes #24 

Added and setup Chakra UI. 
This does not involve the addition of a theme as there is a separate issue for it: #34 

Also changed the index page to use Chakra UI components (with increased font sizes) so that this can be tested:
- Before
  
  <img width="297" alt="image" src="https://user-images.githubusercontent.com/67077705/215923371-a27ab54b-8dec-4b48-8e08-7ee97c3dc7bc.png">

- After
  <img width="319" alt="image" src="https://user-images.githubusercontent.com/67077705/215923231-59fef52e-b653-4563-ba27-112e70d7116f.png">


### Steps for testing
This assumes you already have starchart [setup locally](https://github.com/Seneca-CDOT/starchart/blob/41c1e750936c604fb797b87288dc04a36992b43f/CONTRIBUTING.md#development)

- `npm install`
- `npm run dev`
- navigate to `localhost:3000` and see that the app works and the index page has the updated styles
